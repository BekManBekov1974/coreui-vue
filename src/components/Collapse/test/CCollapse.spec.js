import { mount } from '@vue/test-utils'
import Component from '../CCollapse'

const ComponentName = 'CCollapse'
const defaultWrapper = mount(Component)
const customWrapper = mount(Component, {
  propsData: {
    duration: 500,
    show: true,
    navbar: true,
    toggler: 'some_id'
  },
  slots: {
    default: 'CCollapse content'
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(defaultWrapper.element).toMatchSnapshot()
  })
  it('renders correctly', () => {
    expect(customWrapper.element).toMatchSnapshot()
  })
})
