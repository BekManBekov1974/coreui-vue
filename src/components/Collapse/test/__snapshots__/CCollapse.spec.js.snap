// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CCollapse renders correctly 1`] = `
<div
  class=""
  style="display: none;"
/>
`;

exports[`CCollapse renders correctly 2`] = `
<div
  class="c-navbar-collapse"
>
  CCollapse content
</div>
`;
